lib/openshift/local_volume_local_storage_openshift_io.rb

```diff
@@ -0,0 +1,8 @@
+require 'openshift/project_resource'
+
+module BushSlicer
+  # represents LocalVolume CRD
+  class LocalVolumeLocalStorageOpenshiftIo < ProjectResource
+    RESOURCE = 'localvolume.local.storage.openshift.io'
```

Comment:
I believe this needs to be `localvolumes.local.storage.openshift.io'
I mean the `s` at the end of RESOURCE. Sorry for delay, please ping me if I fail to respond in a few days!

@pruan-rht , @liangxia , this is good to merge on your end?